src/main/resources/theme/leaderboard/leaderboards.fxml

```diff
@@ -16,7 +16,7 @@
       fx:controller="com.faforever.client.leaderboard.LeaderboardsController">
     <Separator minHeight="10.0"/>
     <HBox alignment="CENTER_LEFT" spacing="10.0">
-        <ComboBox fx:id="leaderboardComboBox" onAction="#onLeaderboardSelected"/>
```

Comment:
Why the change from the actionEvent to the listener? I would think the actionEvent would be better and less prone to leaks

Reply:
To simplify the code. We don't need to manually call the `leaderboardComboBox.getValue()` method. The listener will automatically give us the newly selected value. But if this is done to be less prone to leaks, then okay.